src/cpu-template-helper/src/template/dump/aarch64.rs

```diff
@@ -20,38 +32,47 @@ pub fn config_to_template(cpu_config: &CpuConfiguration) -> CustomCpuTemplate {
 #[cfg(test)]
 mod tests {
     use vmm::arch::aarch64::regs::Aarch64Register;
+    use vmm::arch::regs::{Aarch64RegisterVec, RegData128};
 
     use super::*;
 
-    fn build_sample_regs() -> Vec<Aarch64Register> {
-        vec![
-            Aarch64Register {
-                id: 0x0,
-                value: 0xffff_ffff_ffff_ffff_ffff_ffff_ffff_ffff,
-            },
-            Aarch64Register {
-                id: 0xffff_ffff_ffff_ffff,
-                value: 0x0000_ffff_0000_ffff_0000_ffff_0000_ffff,
-            },
-            Aarch64Register {
-                id: 0x1,
-                value: 0x0000_ffff_0000_ffff_0000_ffff_0000_ffff,
-            },
-        ]
+    const KVM_REG_SIZE_U128: u64 = 0x0040000000000000;
+
+    fn build_sample_regs() -> Aarch64RegisterVec {
```

Comment:
will this now only exercise 128-bit regs?

Reply:
Test was using 128 bit regs, so I did not change it. 
I can update it to use different sizes.

Reply:
I think it would be a good idea. We will be confident that they work when we need them at least.

Reply:
change tests to use different sized register